### ISSUE TITLE
start-storybook throws TypeError: storyStore.setSelectionSpecifier is not a function

### ISSUE THREAD

**Issue by Thisen** [storybookjs/storybook#11772]
**Describe the bug**
When I access my local development version of Storybook (`start-storybook ...`), it throws with a TypeError: 
`Uncaught TypeError: storyStore.setSelectionSpecifier is not a function`

**To Reproduce**
1. Run `start-storybook -p 9001`
2. Access Storybook at `localhost:9001`
3. See console with TypeError

**Expected behavior**
That Storybook can open and usable.

**Screenshots**
![image](https://user-images.githubusercontent.com/10807938/89191963-f5e48080-d5a3-11ea-89d8-29a786e1fb32.png)

**Code snippets**
`main.ts:`
```
const path = require("path");

import { Configuration } from "webpack";

module.exports = {
  stories: ["../stories/Introduction.stories.tsx", "../stories/**/*.stories.@(tsx|mdx)"],
  addons: [
    "@storybook/addon-a11y",
    "@storybook/addon-actions",
    "@storybook/addon-docs",
    "@storybook/addon-knobs",
    "@storybook/addon-links",
    "@storybook/addon-backgrounds",
  ],
  logLevel: "debug",
  // eslint-disable-next-line @typescript-eslint/require-await
  webpackFinal: (config: Configuration) => {
    // add monorepo root as a valid directory to import modules from
    config.resolve!.plugins!.forEach((p) => {
      // @ts-ignore
      if (Array.isArray(p.appSrcs)) {
        // @ts-ignore
        p.appSrcs.push(path.join(__dirname, "..", "..", ".."));
      }
    });

    return config;
  },
};
```
**System:**
```
Environment Info:

  System:
    OS: macOS 10.15.6
    CPU: (12) x64 Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
  Binaries:
    Node: 12.16.1 - ~/.nvm/versions/node/v12.16.1/bin/node
    Yarn: 1.22.4 - ~/.yarn/bin/yarn
    npm: 6.13.4 - ~/.nvm/versions/node/v12.16.1/bin/npm
  Browsers:
    Chrome: 84.0.4147.105
    Safari: 13.1.2
  npmPackages:
    @storybook/react: ^6.0.0-rc.24 => 6.0.0-rc.24 
```

**Comment by shilman** [comment]
Did you try removing `node_modules` and/or lockfiles and reinstalling?

And if that doesn't work, do you have a public repro?

**Comment by Thisen** [comment]
It worked after I upgraded to 6.0.0-rc.25.

**Comment by jbmusso** [comment]
TLDR; The good old `rm -rf node_modules` did the trick.

I was using `v6.0.21` and eventually faced the same issue. Upgraded to `v6.0.25` and that didn't fix it.

Since I work in a monorepo with 500k+ packages managed by yarn workspaces, I `rm -rf`'ed the top-level `node_modules` and package-level `node_modules`, reinstalled everything from scratch (with `v6.0.25`), and everything went smoothly.

It could be a side-effect when installing some other package somewhere, but it's hard/time-consuming to track considering our monorepo setup.

**Comment by shilman** [comment]
500k+ packages! 🤯
